frontend/src/context/SaleContext.tsx

```diff
@@ -245,8 +245,8 @@ export const SaleProvider: FC<Props> = ({ children }) => {
 
             resetSaleState();
         } catch (err) {
-            console.log(err);
             createErrorToast(err);
+            throw err;
```

Comment:
We have to throw here so that downstream wrappers (in components) can handle their state accordingly